src/Modal.js

```diff
@@ -0,0 +1,70 @@
+import { classesList } from './classesList';
+import { Wolf } from './wolf/Wolf';
+import { Sheep } from './sheep/Sheep';
+import { Fox } from './fox/Fox';
+import { Antelope } from './antelope/Antelope';
+import { Turtle } from './turtle/Turtle';
+import { Grass } from './grass/Grass';
+import { Guarana } from './guarana/Guarana';
+import { Berry } from './berry/Berry';
+import { Thistle } from './thistle/Thistle';
+
+export class Modal {
+  constructor(board) {
+    this.board = board;
+    this.isVisible = false;
+    this.activeTile = null;
+    this.initializeEventListenerOnModal();
+  }
+  activate = (tile) => {
+    const modal = document.getElementById('addOrganismModal');
+    modal.style.display = 'block';
+    this.isVisible = true;
+    this.activeTile = tile;
+  };
+
+  deactivate = () => {
+    const modal = document.getElementById('addOrganismModal');
+    modal.style.display = 'none';
+    this.isVisible = false;
+    this.activeTile = null;
+  };
+  initializeEventListenerOnModal = () => {
+    const icons = document.querySelectorAll('.icon-container');
+    for (let i = 0; i < icons.length; ++i) {
+      const icon = icons[i];
+      icon.addEventListener('click', this.eventToTrigger);
+    }
+    const closeButton = document.querySelector('#close-button');
+    closeButton.addEventListener('click', this.eventToTrigger);
+  };
+  eventToTrigger = (event) => {
+    this.checkClickOnIcon(event);
+  };
+  checkClickOnIcon = (event) => {
+    const organismIdToClassMapping = {
```

Comment:
```suggestion
    // comment here
    const organismIdToClassMapping = {
```
Please leave a comment here that the keys of this object are corresponding with the id of the animal element in the modal

Reply:
done